app/starship/src/main-entry.js

```
@@ -0,0 +1,178 @@
+    window.fullorg = org
+    updateViewLocation()
+    updatechannelList()
+    updateDirectList()
+    window.wsswitchertippy[0].destroy()
+    var workspacelist = await authClient.organization.list()
+    var workspaceswitcherlist = ""
+    workspacelist.data.forEach(workspace => {
+        if(workspace.id == org.data.id) {
+            var active = "true"
+        } else {
+            var active = "false"
```

Biome (2.1.2)

[error] Shouldn't redeclare 'active'. Consider to delete it or rename it.

'active' is defined here:


(lint/suspicious/noRedeclare)